xbmc/Application.cpp

```diff
@@ -3397,6 +3397,12 @@ void CApplication::StopPlaying()
     int iWin = gui->GetWindowManager().GetActiveWindow();
     if (m_appPlayer.IsPlaying())
     {
+      auto winSystem = CServiceBroker::GetWinSystem();
+      if (nullptr != winSystem)
+      {
+        RESOLUTION_INFO res = winSystem->GetGfxContext().GetResInfo();
+        winSystem->SetFullScreen(true, res, false);
+      }
```

Comment:
Immediately after posting my three changes, my GSoC student (working on Windows 10) came across a strange error. On game exit, the app went fullscreen and this happened:

![image (1)](https://user-images.githubusercontent.com/531482/93358698-a75b1080-f7f6-11ea-9b38-e2803762f412.png)

Commenting this code block fixed his problem. Is there a reason fullscreen is set on exit? Can it be removed?

Reply:
The idea behind setting back to fullscreen while closing app is, in Linux when we run Kodi, it always runs in fullscreen, I did not see window mode under settings.
May be in windows has window mode and this could be handled differently, which I am not aware off.